### PR TITLE
add Singapore

### PR DESCRIPTION
**Pull request by jarek**
See comments in parser script for sources and discussion.

Source for names in other languages is Wikipedia's interwiki links :)

Completes #653

### CODE REVIEW

File: parsers/SG.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+import arrow
+import requests
+
+
+# REMAINING TODO: integrate in other files
```
Comment (corradio) [inline parsers/SG.py]: Is there still something to be done here?

Comment (jarek) [inline parsers/SG.py]: Nope, sorry, please remove this line. I meant the README and the config, but forgot to take it out once done...

**Comment by jarek** [comment]
Ah, now that you mention it, I could have included reported past values for price from the https://www.emcsg.com/marketdata/priceinformation page, but parsing that would have been a bigger pain. (And you said the whole price thing is going to be reworked anyway?) There doesn't appear to be historical-as-in-three-hours-ago data for generation mix, the API endpoint always returns the "current" value.

**Comment by corradio** [comment]
Got it. Yeah pricing is complicated, as it's hard to compare across regions
because all markets have their specificities.

**Pull request merged**